src/main/java/no/unit/nva/model/Contributor.java

```diff
@@ -20,53 +17,26 @@ public Contributor() {
     }
 
     private Contributor(Builder builder) {
-        setArpId(builder.arpId);
-        setOrcId(builder.orcId);
-        setName(builder.name);
-        setNameType(builder.nameType);
-        setImportedAffiliation(builder.affiliation);
+        setIdentity(builder.identity);
+        setAffiliation(builder.affiliation);
         setSequence(builder.sequence);
         setRole(builder.role);
     }
 
-    public String getArpId() {
-        return arpId;
-    }
-
-    public void setArpId(String arpId) {
-        this.arpId = arpId;
-    }
-
-    public String getOrcId() {
-        return orcId;
-    }
-
-    public void setOrcId(String orcId) {
-        this.orcId = orcId;
-    }
-
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
-
-    public String getNameType() {
-        return nameType;
+    public Identity getIdentity() {
+        return identity;
     }
 
-    public void setNameType(String nameType) {
-        this.nameType = nameType;
+    public void setIdentity(Identity identity) {
+        this.identity = identity;
     }
 
-    public List<String> getImportedAffiliation() {
-        return importedAffiliation;
+    public List<Organization> getAffiliation() {
```

Comment:
If affiliations are always plural, we should make this explicit in the nomenclature